src/lib/util.test.js

```diff
@@ -0,0 +1,59 @@
+const should = require(`should`);
```

Comment:
While a `tests/` directory at the root of the project is certainly valid, I tend to prefer `filename.test.js` files living alongside code files. This achieves two things:
- it helps put tests front-and-center
- since tests often help document how something works (or at least what’s expected of it), having them side-by-side with your code files is just convenient

There are cases in which I would argue that a `tests` directory would be better. For example, when a codebase is full of large modules that achieve a lot, requiring multiple tests files for each module; another example would be if you need a lot of integration tests. I don’t think we need integration tests here, really, since this is just a wrapper on a 3rd party library. Likewise, its modules do very little, so needing multiple test files for any single module is highly unlikely. Given these conditions, I think keeping tests beside code files is a good idea.